expenses-manager-frontend/src/main/frontend/src/app/daily-stats/daily-stats.component.css

```diff
@@ -0,0 +1,93 @@
+.main-container {
+    display: flex;
+}
+
+.content-wrapper {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    padding: 20px;
+}
+
+.header {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-bottom: 20px;
+}
+
+.total-amount {
+    display: flex;
+    align-items: center;
+    gap: 15px;
+}
+
+.total-amount h1 {
+    margin: 0;
+    font-size: 36px;
+    font-weight: bold;
+}
+
+.period-tabs {
+    display: flex;
+    justify-content: center;
+    gap: 20px;
+    margin-top: 10px;
+}
+
+.period-tabs a {
+    text-decoration: none;
+    font-size: 18px;
+    color: black;
+    cursor: pointer;
+}
+
+.period-tabs a.active {
+    border-bottom: 2px solid #673ab7; /* Exemplu pentru tab activ */
```

Comment:
Culorile le vom retine in styles.css ca si variable, astfel incat daca va trebui sa schimbam o culoare sa schimbam doar in acel fisier, iar acele il vom folosi ceva de genul: border-bottom: 0.2rem solid var(--primary-color);